dbt_coves/tasks/extract/main.py

```diff
@@ -21,7 +21,7 @@ def register_parser(cls, sub_parsers, base_subparser):
         ext_subparser = sub_parsers.add_parser(
             "extract",
             parents=[base_subparser],
-            help="Extracts data from different systems.",
+            help="Extracts configurations from Airbyte",
```

Comment:
This was correct as it was before "dbt-coves extract" without additional arguments is not related to airbyte yet.

Reply:
@ssassi @noel rolled-back to previous message: `"Extracts data from different systems."`